apps/cowswap-frontend/src/legacy/hooks/usePriceImpact/useFiatValuePriceImpact.ts

```diff
@@ -10,22 +11,37 @@ import { useTradeUsdAmounts } from 'modules/usdAmount'
 
 import { useSafeMemo } from 'common/hooks/useSafeMemo'
 
-const FIAT_VALUE_LOADING_THRESHOLD = ms`0.1s`
+const TRADE_SETUP_DEBOUNCE = ms`3s`
```

Comment:
Is there a chance we just use the state instead of relying on timers? 

Like, until we have the first quote (the optimal, not the fast) and the USD price is loaded (or was attempted and failed to fetch).

For example states:
- we don't know the USD price, we don't know the best quote, we know the fast quote ---> `❌ don't show the WARN`
- we don't know the USD price, we know the best quote --> `❌  we don't show the WARN` 
- we know the USD price, we know the best quote --> `✅ we show the WARN` 
- the USD price failed to load, we know the best quote --> `✅ we show the WARN`

Reply:
thanks for this